frontend/workflows/projectSelector/src/types.tsx

```diff
@@ -106,11 +106,31 @@ export interface DashState {
   projectData: { [projectName: string]: IClutch.core.project.v1.IProject };
 }
 
-export type DashActionKind = "UPDATE_SELECTED";
+export interface TimeData {
+  [eventsKey: string]: IClutch.timeseries.v1.IPoint[];
+}
 
+export interface TimeDataUpdate {
+  key: string;
+  points: IClutch.timeseries.v1.IPoint[];
+}
+
+export interface TimelineState {
+  // Contains a mapping of card names to their event time points
+  // See https://github.com/lyft/clutch/blob/main/api/timeseries/v1/timeseries.proto
+  timeData: TimeData;
+}
+
+export type DashActionKind = "UPDATE_SELECTED";
```

Comment:
same nit as above. can dash things be kept grouped together? for easier discoverability and readability